src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -73,6 +73,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case FindCommand.COMMAND_WORD:
             return new FindCommandParser().parse(arguments);
 
+        case ListCommand.COMMAND_WORD:
+            return new ListCommand();
+
```

Comment:
I think this may be removed as it is handled above

Reply:
fixed as well